package.json

```diff
@@ -123,6 +123,7 @@
     "@percy/storybook": "^5.0.1",
     "@skyscanner/bpk-foundations-web": "^17.9.0",
     "@skyscanner/bpk-svgs": "^19.1.0",
+    "@skyscanner/eslint-config-skyscanner": "^17.4.0",
```

Comment:
Ah this is actually also pulled in via `stylelint-config-skyscanner` so it might be worth us bumping in there and then bumping the below dep up a version to avoid multiple versions of e-c-s being installed and conflicting

Reply:
Yeah, good point. I think everything will still shake out to the same version due to the `^`. In `main` it's the same version, do you know why it's a dependency and not a dev-dep? https://github.com/Skyscanner/stylelint-config-skyscanner/blob/76c753b382d9a608a569f8efde0f75873197dab5/package.json#L40

Reply:
```
└─┬ @skyscanner/stylelint-config-skyscanner@10.2.0
  └── @skyscanner/eslint-config-skyscanner@16.3.0
```
Oh yeah, it's actually quite out of date.

Reply:
Ah, found this reasoning: https://github.com/Skyscanner/stylelint-config-skyscanner/blob/76c753b382d9a608a569f8efde0f75873197dab5/docs/eslint-as-dependency.md?plain=1#L3

Reply:
I've updated stylelint to latest, 

This results in this tree (no dupe)
```
├── @skyscanner/eslint-config-skyscanner@17.4.0
└─┬ @skyscanner/stylelint-config-skyscanner@11.1.0
  └── @skyscanner/eslint-config-skyscanner@17.4.0 deduped
```

Reply:
I interpreted the doc as the reason that stylelint consumes eslint-config, not that we shouldn't install the eslint-config if we consume it. 

> ...allowing consumers to update to the latest of each and have a tree that is in sync.

This reads, to me, like we _should_ consume both. It's, IMO, not right to rely on a transitive dependency. Many (all the ones I've checked) of our consumers use both stylelint-config and eslint-config alongside one another.